lib/matplotlib/axes/_axes.py

```diff
@@ -6457,7 +6457,21 @@ def hist2d(self, x, y, bins=10, range=None, normed=False, weights=None,
 
         Other parameters
         ----------------
-        kwargs : :meth:`pcolorfast` properties.
+        cmap : [None, Colormap, string], optional
```

Comment:
@dstansby see section 4: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt

if anything, they should be `{}`'s